build/linux/installer/datafiles/base_container.data

```diff
@@ -21,12 +21,15 @@ MAINTAINER:              'Microsoft Corporation'
 /etc/opt/microsoft/docker-cimprov/container.conf;			    build/linux/installer/conf/container.conf;                      644; root; root
 
 /opt/fluent-bit/bin/out_oms.so;                                       intermediate/${{BUILD_CONFIGURATION}}/out_oms.so;  	755; root; root
+/opt/fluent-bit/bin/containerinventory.so;                                       intermediate/${{BUILD_CONFIGURATION}}/containerinventory.so;  	755; root; root
```

Comment:
curios, adding perf and containerinventory so files, do we see increasing in the container image size?

Reply:
both of them are about 13MB so total increase by 26MB approx